autoconnect/autoconnect-ws/autoconnect-ws-sm/src/unidentified.rs

```diff
@@ -93,6 +95,26 @@ impl UnidentifiedClient {
             })
             .send();
 
+        // This is the first time that the user has connected "today".
+        if flags.user_gm {
+            // Return the number of channels for the user using the internal channel_count.
+            // NOTE: this metric can be approximate since we're sampling to determine the
+            // approximate average of channels per user for business reasons.
+            self.app_state
+                .metrics
+                .incr_with_tags("ua.connection.check")
+                // TODO: These should reflect the actual os and browser, but getting those
+                // values at this point would be very complicated. For now, just use "desktop"
+                .with_tag("os", "desktop")
+                .with_tag("browser", "desktop")
+                .send();
```

Comment:
Yeah, it might be a bit extra. The thought was that since the histogram is more about collecting `channel_count()` having a different one that noted the check might capture instances where `channel_count()` was zero. I'm guessing it's probably not that much of a concern, so I'm fine dropping it.